lib/rubocop/cop/root_cops/factory_file_name.rb

```diff
@@ -1,25 +1,67 @@
-require "pry"
+require "active_support/inflector"
 
 module RuboCop
   module Cop
     module RootCops
       class FactoryFileName < Cop
-        FILE_NAME_ERROR = "Factory name should match file name".freeze
+        # Whitelisted words feel like invariant nouns but they are really not
+        # Ex: the singular of data is datum, but we should be able to name a factory '*_data' without error
+        WHITELIST = ["data".freeze, "metadata".freeze].freeze
```

Comment:
I feel like we shouldn't have this within the cop. `ActiveSupport` provides a way to provide irregular inflections and we should let the applications using this gem configure that rather than hard coding these here. See: https://github.com/Root-App/root-server/blob/d89db81dd0b8b6bd2c3a6d7e49ed4dd5d51b37ac/config/initializers/inflections.rb#L18-L24

Reply:
That being said, maybe double check that the inflection config is picked up when running rubocop on root-server?

Reply:
>  See: https://github.com/Root-App/root-server/blob/d89db81dd0b8b6bd2c3a6d7e49ed4dd5d51b37ac/config/initializers/inflections.rb#L18-L24

This is awesome! These are the exact use cases I was running into.